.github/workflows/build_playground_backend.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         uses: google-github-actions/setup-gcloud@v0
         with:
             install_components: 'beta,cloud-datastore-emulator'
+            version: '389.0.0'
```

Comment:
It was necessary because the last version causes some bad effects, e.g. it requires Java 11+ in lieu of Java 8